libs/core/langchain_core/tools/base.py

```diff
@@ -399,9 +398,9 @@ class ChildTool(BaseTool):
     You can use these to eg identify a specific instance of a tool with its use case.
     """
 
-    handle_tool_error: Optional[Union[bool, str, Callable[[ToolException], str]]] = (
-        False
-    )
+    handle_tool_error: Union[
```

Comment:
i think it is confusing to put a tuple of exceptions to handle into `handle_tool_errors` -- currently it defines how we're populating the `ToolMessage.content`, so it would be unintuitive to me as a user that i would need to pass exceptions here (as opposed to a separate kwarg). besides, what if i want to customize the `ToolMessage.content` AND the exceptions that are handled?

Reply:
then you use the Callable